docs/source/usage/build-public.rst

```diff
@@ -10,8 +10,8 @@ build-public
 
     This command creates a redacted version of an SBOM fit for publication. It:
 
-    * can optionally delete entire components matching a JSON schema provided by the user, and it
     * deletes any *property* (i.e., item in the ``properties`` array of a component) whose name starts with ``internal:`` from all components.
+    * can optionally delete entire components matching a JSON schema provided by the user, and it
```

Comment:
Now the "and it" at the end of the sentence does not fit, please either write the "and it" at the end of the first bullet point or remote it completely.